src/test/java/uk/gov/companieshouse/extensions/api/attachments/AttachmentDTOUnitTest.java

```diff
@@ -1,21 +1,23 @@
 package uk.gov.companieshouse.extensions.api.attachments;
 
-import static org.junit.Assert.assertEquals;
+
 
 import java.io.IOException;
 
 import com.fasterxml.jackson.databind.ObjectMapper;
 
-import org.junit.Test;
-import org.junit.experimental.categories.Category;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
 import org.springframework.web.multipart.MultipartFile;
 
 import uk.gov.companieshouse.extensions.api.Utils.Utils;
-import uk.gov.companieshouse.extensions.api.groups.Unit;
 import uk.gov.companieshouse.extensions.api.requests.ExtensionsLinkKeys;
 import uk.gov.companieshouse.service.links.Links;
 
-@Category(Unit.class)
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+
```

Comment:
Could you remove one of these empty lines?

Reply:
Yes I can.